src/garage/tf/algos/ddpg.py

```diff
@@ -229,6 +235,7 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         """Object.__setstate__."""
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
we should probably keep this rule on. please only do per-line disables, not per-file or per-function (unless it's an exceptional circumstance).

this stanza appears frequently when picking. we could either do a per-line disable for these methods, or change this to

```python
def __setstate__(self, state):
    self.__dict__.clear()   # is this needed on top of update()?
    self.__dict__.update(state)
```

@krzentner WDYT?